packages/storefront-lwc/src/modules/commerce/productDetail/__tests__/productDetail.test.js

```diff
@@ -0,0 +1,326 @@
+import { createElement } from 'lwc';
+import {
+    mockQuery,
+    mockMutation,
+    reset,
+    getLastMutation,
+} from '@lwce/apollo-client';
+import ProductDetail from 'commerce/productDetail';
+
+import { registerWireService } from '@lwc/wire-service';
+import { register } from 'lwc';
+
+registerWireService(register);
+
+function tick(time = null) {
```

Comment:
This is a helper function to wait for a period of time. Usually a `time` param should _not_ be passed. When there is no parameter, it just skips a tick of the event loop, so the test still goes fast but allows for an asynchronous operation to take place.